lib/vm.py

```diff
@@ -474,7 +474,9 @@ def destroy_vtpm(self):
         return self.host.xe('vtpm-destroy', {'uuid': vtpm_uuid}, force=True)
 
     def clone(self):
-        name = self.name() + '_clone_for_tests'
+        name = self.name()
+        if not name.endswith('_clone_for_tests'):
+            name += '_clone_for_tests'
```

Comment:
I'd rather not merge this change until a better handling is done, otherwise cloned VMs created from cached VMs will be hard to differentiate from cache by their name.

Reply:
yes there is a FIXME for this already

Reply:
I didn't know if the FIXME was meant for before or after the merge :)

Reply:
inside commit message is explicitly *before* ;)

Reply:
done in last push